lib/src/fade.dart

```diff
@@ -5,12 +5,15 @@ import 'animated_text.dart';
 ///
 /// ![Fade example](https://raw.githubusercontent.com/aagarwal1012/Animated-Text-Kit/master/display/fade.gif)
 class FadeAnimatedText extends AnimatedText {
-  FadeAnimatedText(
-    String text, {
-    TextAlign textAlign = TextAlign.start,
-    required TextStyle textStyle,
-    Duration duration = const Duration(milliseconds: 2000),
-  }) : super(
+  double fadeInEnd, fadeOutStart;
+
+  FadeAnimatedText(String text,
+      {TextAlign textAlign = TextAlign.start,
+      required TextStyle textStyle,
+      Duration duration = const Duration(milliseconds: 2000),
+      this.fadeInEnd = 0.5,
+      this.fadeOutStart = 0.8})
```

Comment:
Certain labels are often paired, like:
* _Begin_ and _End_
* _Start_ and _Finish_

You are using _End_ for `fadeInEnd` which is normally paired with _Begin_, and then `Start` for `fadeOutStart`.  Since these values pertain to the `begin` and `end` of an [Interval](https://api.flutter.dev/flutter/animation/Interval-class.html), I suggest keeping to those labels.

As a result, I recommend renaming `fadeOutStart` to be `fadeOutBegin`.

Reply:
Done